src/main/resources/avro/referencemethods.avdl

```diff
@@ -325,4 +390,185 @@ GetSequenceBasesResponse getSequenceBases(
+  */
+  union { null, string } referenceSetId = null;
+
+  /**
+  If not null, return only `Join`s which belong to this variant set.
+  
+  If null, `referenceSetId` must not be null.
+  */
+  union { null, string } variantSetId = null;
+
+  /**
+  If not null, return only `Join`s with at least one `Side` on this sequence.
+  sequences.
+
+  If null, `start`, `length`, and `strand` must be null.
+  */
+  union { null, string } sequenceId = null;
+
+  /**
+  If not null, return only `Join`s with at least one `Side` at this index or
+  later in the `Sequence` specified by `sequenceId`.
+
+  If null, `length` must be null.
+  */
+
+  union { null, long } start = null;
+
+  /**
+  If not null, return only `Join`s with at least one `Side` on the sequence
+  specified by `sequenceId` in the interval [`start`, `start` + `length`).
+  */
+  union { null, long } length = null;
+
+  /**
+  If not null, return only `Join`s which join onto this strand of the `Sequence`
+  specified by `sequenceId`.
+  */
+  union { null, org.ga4gh.models.Strand } strand = null;
+
+  /**
+  Specifies the maximum number of results to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  To get the next page of results, set this parameter to the value of
+  `nextPageToken` from the previous response.
+  */
+  union { null, string } pageToken = null;
+}
+
+/**
+This is the response from `POST /joins/search` expressed as JSON.
+*/
+record SearchJoinsResponse {
+  /** The list of matching joins. */
+  array<org.ga4gh.models.Join> joins = [];
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  Provide this value in a subsequent request to return the next page of
+  results. This field will be empty if there aren't any additional results.
+  */
+  union { null, string } nextPageToken = null;
+}
+
+/**
+Gets a list of `Join` matching the search criteria.
+
+`POST /joins/search` must accept a JSON version of
+`SearchJoinsRequest` as the post body and will return a JSON
+version of `SearchJoinsResponse`.
+*/
+SearchJoinsResponse searchJoins(
+  /**
+  This request maps to the body of `POST /joins/search`
+  as JSON.
+  */
+  SearchJoinsRequest request) throws GAException;
+
+
+/****************  /subgraph/radius  *******************/
+/**
+This request maps to the body of `POST /subgraph/radius` as JSON. Specifies
+a`Position` and a radius (in bases), and requests the subgraph of all `Join`s
+and `Sequence`s reachable within that number of bases from that position.
+
+Starting at the specified `Position`, and with a 0 radius denoting only that
+`Position`, walk outwards this many bases along all possible paths, traversing
+`Join`s only if necessary. All `Sequences` visited during this walk, and all
+`Join`s traversed, should be returned.
+*/
+record SubgraphRadiusRequest {
```

Comment:
Ah, I see... I tried generating a `_protocol_definitions.py` module from this code, and was missing the `subgraphRadius()` method from it because it (and its associated request/response objects) doesn't fit the typical `/something/search` pattern that the `process_schemas.py` relies on (will need to raise issue in server project to fix that script)